cmd/api/main.go

```diff
@@ -59,6 +65,7 @@ func NewServer(k8sClient kubernetes.Interface, controller routers.Controller) *S
 }
 
 func main() {
+	log.Printf("Starting KubeArchive API with version '%s', commit SHA '%s', built at '%s'", version, commit, date)
```

Comment:
"built at" would indicate a location, for a date I think this should be just "built".

```suggestion
	log.Printf("Starting KubeArchive API with version '%s', commit SHA '%s', built '%s'", version, commit, date)
```